core/src/main/java/bio/terra/pearl/core/service/kit/LivePepperDSMClient.java

```diff
@@ -14,35 +14,41 @@
 import org.springframework.web.reactive.function.client.WebClient;
 import reactor.core.publisher.Mono;
 
+import javax.validation.Validator;
 import java.time.Instant;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.UUID;
 import java.util.function.Function;
+import java.util.stream.Collectors;
 
 @Component
 public class LivePepperDSMClient implements PepperDSMClient {
     private final PepperDSMConfig pepperDSMConfig;
     private final WebClient webClient;
     private final ObjectMapper objectMapper;
+    private final Validator validator;
 
     public LivePepperDSMClient(PepperDSMConfig pepperDSMConfig,
                                WebClient.Builder webClientBuilder,
-                               ObjectMapper objectMapper) {
+                               ObjectMapper objectMapper,
+                               Validator validator) {
         this.pepperDSMConfig = pepperDSMConfig;
         this.webClient = webClientBuilder.build();
         this.objectMapper = objectMapper;
+        this.validator = validator;
     }
 
     @Override
-    public String sendKitRequest(Enrollee enrollee, KitRequest kitRequest, PepperKitAddress address) {
-        var request = buildAuthedPostRequest("shipKit", makeKitRequestBody(enrollee, kitRequest, address));
+    public String sendKitRequest(String studyShortcode, Enrollee enrollee, KitRequest kitRequest, PepperKitAddress address) {
+        var request = buildAuthedPostRequest("shipKit", makeKitRequestBody(studyShortcode, enrollee, kitRequest, address));
         return retrieveAndDeserializeResponse(request, String.class);
     }
 
     @Override
     public PepperKitStatus fetchKitStatus(UUID kitRequestId) {
-        var request = buildAuthedGetRequest("kitStatus/kit/%s".formatted(kitRequestId));
+        // TODO: change "juniperKit" to "juniperkit" after DSM updates this path
```

Comment:
Pegah is going to update this, but it's a code change that will need to be made/reviewed/deployed to DSM.